packages/weather/OpenWeatherMap.py

```diff
@@ -0,0 +1,218 @@
+                    "acquiring",
+                    utils.translate("acquiring")
+                )
+
+                payload["city"] = item["sourceText"].title()
+                payload["observation"] = payload["owm"].weather_at_place(payload["city"])
+                payload["wtr"] = payload["observation"].get_weather()
+
+                return func(payload)
+        return utils.output(
+            "end",
+            "request_error",
+            utils.translate("request_error")
+        )
+
+    return wrapper_acquire_weather
+
+
+# Methods
+
+@load_config
+@acquire_weather
+def current_weather(payload):
+    """
+    Get the current weather.
+    """
+
+    detailed_status = payload["wtr"].get_detailed_status()
+    temperatures = payload["wtr"].get_temperature(payload["temperature_units"])
+    humidity = payload["wtr"].get_humidity()
+
+    return utils.output(
+        "end",
+        "current_weather",
+        utils.translate(
+            "current_weather",
+            {
+                "detailed_status": detailed_status.capitalize(),
+                "city": payload["city"],
+                "temperature": temperatures["temp"],
+                "temperature_units": payload["temperature_units"].capitalize(),
+                "humidity": humidity
+            }
+        )
+    )
+
+
+@load_config
+@acquire_weather
+def temperature(payload):
+    """
+    Get the current temperature.
+    """
+
+    temperatures = payload["wtr"].get_temperature(payload["temperature_units"])
+
+    return utils.output(
+        "end",
+        "temperature",
+        utils.translate(
+            "temperature",
+            {
+                "city": payload["city"],
+                "temperature": temperatures["temp"],
+                "temperature_units": payload["temperature_units"].capitalize()
+            }
+        )
+    )
+
+
+@load_config
+@acquire_weather
+def humidity(payload):
+    """
+    Get the current humidity.
+    """
+
+    humidity = payload["wtr"].get_humidity()
+
+    return utils.output(
+        "end",
+        "humidity",
+        utils.translate(
+            "humidity",
+            {
+                "city": payload["city"],
+                "humidity": humidity
+            }
+        )
+    )
+
+
+@load_config
+@acquire_weather
+def wind(payload):
+    """
+    Get the current wind speed and direction.
+    """
+
+    wind = payload["wtr"].get_wind(payload["wind_speed_units"])
+
+    return utils.output(
+        "end",
+        "wind",
+        utils.translate(
+            "wind",
+            {
+                "city": payload["city"],
+                "wind_speed": wind["speed"],
+                "wind_speed_units_response": payload["wind_speed_units_response"],
+                "wind_direction": wind["deg"]
+            }
+        )
+    )
+
+
+@load_config
+@acquire_weather
+def sunrise(payload):
+    """
+    Get when the sun rises.
+    """
+
+    dt = payload["wtr"].get_sunrise_time("date")
```

Comment:
We should avoid abbreviations in variable names.
So instead to call it `dt`, we could call it `date_time` for example.